sgl-kernel/tests/test_cublas_grouped_gemm.py

```diff
@@ -11,39 +10,40 @@ def torch_grouped_gemm(a_array, b_array, out_dtype):
     return c_array
 
 
-class TestGroupedGemm(unittest.TestCase):
-    def _test_accuracy(self, Ms, Ns, Ks, out_dtype):
-        group_count = len(Ms)
-        a_array = []
-        b_array = []
-        c_array_cublas = []
-        for i in range(group_count):
-            M, N, K = Ms[i], Ns[i], Ks[i]
-            a_array.append(torch.randn((M, K), device="cuda", dtype=out_dtype) * 5)
-            b_array.append(torch.randn((N, K), device="cuda", dtype=out_dtype) * 5)
-            c_array_cublas.append(torch.empty((M, N), device="cuda", dtype=out_dtype))
+# skip if CUDA is not available or CUDA < 12.5
+skip_condition = not torch.cuda.is_available() or (
+    torch.version.cuda is None
+    or tuple(map(int, torch.version.cuda.split("."))) < (12, 5)
+)
+
+
+@pytest.mark.skipif(
+    skip_condition, reason="CUDA not available or CUDA version lower than 12.5"
+)
+@pytest.mark.parametrize("out_dtype", [torch.float16, torch.bfloat16])
+def test_grouped_gemm_accuracy(out_dtype):
+    Ms = [1, 16, 32, 256, 1024]
```

Comment:
For different shapes you should use `@pytest.mark.parametrize` instead of for loop